gradle.properties

```diff
@@ -1,11 +1,11 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties from https://fabricmc.net/develop
-minecraft_version=1.19
+minecraft_version=1.19.1
 yarn_mappings=1.19+build.4
```

Comment:
`.19.1+build.3`

Reply:
Resolved, but I want to add one more thing so don’t merge it just yet